.github/workflows/buildpush.yaml

```diff
@@ -0,0 +1,78 @@
+---
+name: Build and Push Images
+
+on:
+  push:
+
```

Comment:
Skip if the repo is not ramendr/recipe

Reply:
Did this but in a different way. Just need the vars.PUBLISH_IMAGES set to be true. This allows the contributors to have it set along with the other vars if they want to push to their fork of quay automatically on push.

Reply:
allowing github.event_name == 'pull_request' may push from a pull request which is most likely not what you want.

Not checking the branch is also likely not what you want, unless the image include the branch name, but we don't include the branch name.

Reply:
On line 5, I restrict the action to only push and not pull_request. We do include the branch name is the image name in the tag section. For example, for the release-4.16 branch we will have quay.io/ramendr/recipe-bundle:4.16-latest